taxi/models.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=80)
+
+    class Meta:
+        verbose_name = "manufacturer"
+        verbose_name_plural ="manufacturers"
+
+    def __str__(self):
+        return self.name
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+    )
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name="cars")
+
+    class Meta:
+        verbose_name = "car"
+        verbose_name_plural ="cars"
+
+    def __str__(self):
+        return f"Model:{self.model},{self.manufacturer}"
```

Comment:
Avoid overusing f-strings. According to the checklist, you should not use f-strings when they are not necessary. Consider returning the string without using f-string formatting.